src/pages/host/HostCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { ApolloError } from "apollo-client";
+import { Skeleton } from "antd";
+import { ErrorWrapper } from "components/ErrorWrapper";
+import Card from "@leafygreen-ui/card";
+import styled from "@emotion/styled/macro";
+import { css } from "@emotion/core";
+
+interface Props {
+  error: ApolloError;
+  loading?: boolean;
+  metaData: boolean;
+}
+interface StylingProps {
+  metaData: boolean;
+}
+
+export const HostCard: React.FC<Props> = ({
+  children,
+  error,
+  loading,
+  metaData,
+}) => (
+  <SiderCard metaData={metaData}>
+    {loading && !error && (
+      <Skeleton active title={false} paragraph={{ rows: 4 }} />
+    )}
+    {error && !loading && (
+      <ErrorWrapper data-cy="metadata-card-error">{error.message}</ErrorWrapper>
+    )}
```

Comment:
i don't think loading and error are ever truthy at the same time, so `loading && <Component/>` and `error && <Component/>` should be enough

Reply:
Where are you worried about loading and error both being true at the same time? It seems to be [loading && !error], [error && !loading], [!loading && !error].